src/main/scala/viper/gobra/ast/frontend/PrettyPrinter.scala

```diff
@@ -229,6 +229,7 @@ class DefaultPrettyPrinter extends PrettyPrinter with kiama.output.PrettyPrinter
       case PDeferStmt(exp) => "defer" <+> showExpr(exp)
       case PBlock(stmts) => block(showStmtList(stmts))
       case PSeq(stmts) => showStmtList(stmts)
+      case POutline(body, spec) => showSpec(spec) <> "outline" <+> "(" <> line <> showStmtList(body) <> ")"
```

Comment:
you can replace `"(" <> line <> showStmtList(body) <> ")"` by `parens(line <> showStmtList(body))`